targets/runtime.project.json

```diff
@@ -7,26 +7,31 @@
   "frameworks": {
     "dnxcore50": {
       "dependencies": {
```

Comment:
No, I was just blind to the header-only line because the diff was too big for GitHub to display in the browser.  Never mind me . . .